File: year2020/month10/day1005/ms_1005.cpp
#include <iostream>

using namespace std;

int sequenceSum(int value, int by);

int main() {
	int n;

	cin >> n;

	if (0 < n && n <= 100)
		cout << sequenceSum(1, n);

	return 0;
}

int sequenceSum(int value, int by) {
	if (value == by)
		return by;
	else
		return value + sequenceSum(value + 1, by);
}

File: year2020/month10/day1005/ms_1005.cs
using System;

namespace algorithm_71
{
    class ms_1005
    {
        static int SequenceSum(int n) 
        {
            if(n <= 0)
                return 0;
            else
                return n + SequenceSum(n-1);
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n && n <= 100)
                Console.WriteLine(SequenceSum(n));
        }
    }
}
